content/blog/pulumi-kubernetes-operator-2-0/index.md

```diff
@@ -0,0 +1,269 @@
+by the 'restricted' profile of the [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/).
+If necessary, you may configure a stack to use the 'baseline' profile.
+
+A Kubernetes `ServiceAccount` provides an identity for your workspace pod, and we recommend that you create
+a service account for your stack rather than using the `default` account. If your Pulumi program uses the
+Kubernetes provider to manage resources within the cluster, the stack's service account will need permissions, e.g.
+a `RoleBinding` to the `ClusterRole` named `cluster-admin`. See ["Service Accounts"](https://kubernetes.io/docs/concepts/security/service-accounts/) for more information.
+
+The workspace pod has a RPC endpoint that is used by the operator to run stack operations. The pod protects the
+RPC endpoint using Kubernetes RBAC, and for that reason you must bind the `ClusterRole` named `system:auth-delegator` to the
+stack's service account.
+
+<!--more-->
+
+Here's an example of a service account and associated role binding for use by a given stack:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: my-program
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: my-program:system:auth-delegator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:auth-delegator
+subjects:
+- kind: ServiceAccount
+  name: my-program
+  namespace: default
+---
+apiVersion: pulumi.com/v1
+kind: Stack
+metadata:
+  name: my-program
+  namespace: default
+spec:
+  serviceAccountName: my-program
+  ...
+```
+
+### Scalability
+
+Each stack is run in its own "workspace" pod, and so the system scales horizontally to support many stacks.
+The workspace pod is, by default, configured to use minimal resources when idle, and to 'burst' into the unused
+memory and compute resources of the node during a stack operation. You may customize the resource constraints to
+fit the needs of your stack. If your Kubernetes cluster has high utilization, we recommend increasing the resource requests
+and limits. See ["Pod Quality of Service Classes"](https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/) for more information.
+
+The pod is not discarded at the end of a stack operation, it remains in-place unless the stack is deleted.
+We're considering introducing the option to clean up the workspace pod after each operation, to enable you to make a trade-off
+decision between performance and efficiency.
+
+### Robustness
+
+A Kubernetes cluster is a dynamic environment where a given pod may be terminated at any time, e.g. due to maintenance operations,
+unplanned node events, or resource pressure. This can cause problems for ongoing stack operations, and historically
+may cause a stack to become "locked" (thus necessitating the use of `pulumi cancel`).
```

Comment:
```suggestion
unplanned node events, or resource pressure. This can cause problems for ongoing stack operations, and historically
the operator could cause stacks to become "locked", thus necessitating manual intervention with the use of `pulumi cancel`.
```